website/pizzas/api/v2/admin/urls.py

```diff
@@ -0,0 +1,39 @@
+"""Events app API v2 urls."""
+from django.urls import path
+
+from events.api.v2.admin.views import EventRegistrationAdminFieldsView
+from pizzas.api.v2.admin.views import (
+    FoodEventAdminListView,
+    FoodEventAdminDetailView,
+    FoodOrderAdminDetailView,
+    FoodEventOrdersAdminListView,
+    ProductsAdminListView,
+    ProductAdminDetailAPIView,
+)
+
+app_name = "pizzas"
```

Comment:
This is "food" in the non-admin urls, so I'm guessing one of the two should change.